maps/tether/tether-01-surface1.dmm

```diff
@@ -13942,15 +13948,20 @@
 /obj/item/stack/material/phoron,
 /obj/item/stack/material/phoron,
 /obj/machinery/reagentgrinder,
+/obj/structure/extinguisher_cabinet{
+	dir = 8;
+	icon_state = "extinguisher_closed";
+	pixel_x = 30;
+	tag = "icon-extinguisher_closed (WEST)"
```

Comment:
Tags are basically bad, but DreamMaker loves to add them if you use certain combinations of 'Generate instances from directions' or '...icon_states'. It makes all those objects the same, like anything with the same tag is considered the same object in some cases, and can cause weirdness.

Tags also make objects immune to garbage collection which is bad, because tags are considered a self-reference, so their refcount will never drop to 0.

Reply:
You got it, bossfox

Reply:
Doing a search, I found several other "tag =" items on this level. want me to clean those out too? Some atmos equipment, guest pass printers, etc

Reply:
Hmm. No, I'll make a separate PR and remove all of those. There's always the chance that something actually relies on the tag functionality and I wanna check first. Just remove these two for now.

Reply:
Got it! Thank you.